package.json

```diff
@@ -123,6 +125,8 @@
     "uuid": "7.0.3"
   },
   "peerDependencies": {
+    "@types/styled-components": "4.1.8",
+    "@types/warning": ">=3.0.0",
     "react": ">=16.8.0",
     "styled-components": ">=4.3.2"
```

Comment:
I assume the `@types/styled-components` v4.18 is the last one without the issue, but is the `styled-components` v4.3.2 okay with this or is it already checked?

Reply:
Should be ok, but has not actually been tested with current setup. Types are broken since v.4.1.9. Styled-components did work with current develop until all deps were updated and the API's have not changed.